guidelines/mwm-auth.md

```diff
@@ -0,0 +1,86 @@
+<!--
```

Comment:
Is it possible to provide some instructions on how to set up Keycloak? (links to Keycloak site are fine too)

Reply:
Not sure this is a good idea locations of getting started/tutorials can change. I think its fair to assume people can google "keycloak getting started".

If you still think I should put it in I'm open too it.